package.json

```diff
@@ -20,6 +20,7 @@
     "test:pact:run-and-publish": "yarn test:pact && yarn publish-pact",
     "sonar-scan": "sonar-scanner",
     "test:functional": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs --config ./src/test/functional/codecept.conf.js run-workers 2 parallel",
```

Comment:
I've been seeing the following message in the logs when install packages "warning " > codeceptjs@3.3.0" has unmet peer dependency "faker@^5.5.1".

Could you please add faker to the package list unless there is a reason you can't?

Reply:
will do in the next patchset